src/components/20-molecules/top-content-bar/story.js

```diff
@@ -86,6 +75,6 @@ export const TopContentBar = () => {
   `;
 
   render(template, wrapper);
-  wrapper.querySelector('axa-top-content-bar').closed = barClosed;
+  wrapper.querySelector('axa-top-content-bar').closed = false;
```

Comment:
Is this needed? AFAIK the closed property is initialized as falsy already, cf. https://github.com/axa-ch-webhub-cloud/pattern-library/blob/79bcd2e2aee9bf83469004ef251b6222b09d22cb/src/utils/with-react.js#L70.

Reply:
If not shown to be needed, I'd suggest to delete this line.

Reply:
Deleted